charts/core/README.md

```diff
@@ -88,6 +88,7 @@ Chart container component.
 | --------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------ | ------- |
 | `onChartReady`              | Callback to be called when chart is finished rendering.                                                                                       | `() => void`                   |         |
 | `onZoomSelect` _(optional)_ | Callback to be called when a user clicks and drags on a chart to zoom. Click and drag action will only be enabled if this handler is present. | `(ZoomSelectionEvent) => void` |         |
+| `groupId` _(optional)_      | Charts with the same `groupId` will have their tooltips synced across charts.                                                                 | `string`                       |         |
```

Comment:
`groupId` is what it's called in ECharts. We could in theory call it whatever we want though. I was thinking like `syncId` but wasn't sure if that was clearer. Open to ideas here!

Reply:
Since ECharts calls it `groupId` i think the name makes sense. Could groupId be used for anything else?

Reply:
I don't believe so. At least at the moment that's it. Perhaps in the future we could sync more in a group, but nothing is needed with it atm that I'm aware of